src/AppInstallerCommonCore/Public/AppInstallerMsixInfo.h

```diff
@@ -34,6 +36,11 @@ namespace AppInstaller::Msix
         IStream* inputStream,
         IAppxManifestReader** reader);
 
+    // Function to create a stream from a uri string (url address or file system path)
+    void GetStreamFromURI(

```

Comment:
This should return the `ComPtr<IStream>` rather than using an output parameter.  Also, it should probably be a free function somewhere else rather than attached to this type.

Reply:
Updated signature and moved to `AppInstallerDownloader.h`.